testlapack/dlaqr1.go

```diff
@@ -0,0 +1,91 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/floats"
+)
+
+type Dlaqr1er interface {
+	Dlaqr1(n int, h []float64, ldh int, sr1, si1, sr2, si2 float64, v []float64)
+}
+
+func Dlaqr1Test(t *testing.T, impl Dlaqr1er) {
+	rnd := rand.New(rand.NewSource(1))
+
+	for _, n := range []int{2, 3} {
```

Comment:
I was wondering why it was spelled `for _, n := range []int{2, 3} {` instead of `for n := 2; n <= 3; n++ {`.

Now that I write out the alternative. I am happy what you have here was the correct choice.